package.json

```diff
@@ -32,9 +49,12 @@
     "electron-builder": "^24.6.4",
     "eslint": "^8.52.0",
     "eslint-plugin-react": "^7.33.2",
+    "husky": "^8.0.3",
+    "lint-staged": "^15.0.2",
     "next": "^12.3.4",
     "nextron": "^8.12.0",
     "postcss": "^8.4.31",
+    "prettier": "^3.0.3",
```

Comment:
please place it in devDependenceis

Reply:
yes its under devDependencies
<img width="256" alt="image" src="https://github.com/ArweaveTeam/arweave-miner-ui/assets/60331864/fb3e6613-703d-44aa-bc48-432db63c8451">